fdbserver/RestoreUtil.h

```diff
@@ -34,6 +34,12 @@
 #include <cstdint>
 #include <cstdarg>
 
+#define FR_DEBUG_SEVERITY 		  SevDebug
+#define FR_VERBOSE_DEBUG_SEVERITY SevDebugVerbose
+
+#define FRTraceEvent(...) TraceEvent(FR_DEBUG_SEVERITY, "FastRestoreDebug", UID())
+#define FRVBTraceEvent(...) TraceEvent(FR_VERBOSE_DEBUG_SEVERITY, "FastRestoreDebugVerbose", UID())
```

Comment:
A few thoughts:

1. These take arguments but don't use them. Should they just take none?
2. I personally would prefer that the names of these more clearly communicated their intent to be used for debugging. For example, `FrDebugTraceEvent` and `FrVerboseTraceEvent`. 
3. Perhaps an extension of the second point, but are there plans to use these more liberally? With only one event each, it seems like it would be easier and even more clear if the trace events were written as normal rather than using the macros. If you are using them a lot, I could see the argument either way.

Reply:
Good point! 
Another approach is to only define the specific severity level, e.g., `FR_VERBOSE_DEBUG_SEVERITY`, for the fast restore. 
It can be one macro-variable per event (or per node type). 
We will just use the TraceEvent as usual. 

What do you think?